src/test/scala/org/broadinstitute/dsde/rawls/graph/OrientDbTestFixture.scala

```diff
@@ -271,8 +271,8 @@ trait OrientDbTestFixture extends BeforeAndAfterAll {
   }
 
   def withWorkspaceContext[T](workspace: Workspace, writeLock: Boolean, txn: RawlsTransaction)(testCode: WorkspaceContext => T) = {
-    val workspaceContext = workspaceDAO.loadContext(workspace.toWorkspaceName, txn).getOrElse(throw new RawlsException(s"Unable to load workspaceContext for ${workspace.toWorkspaceName}"))
-    txn.withLock(workspaceContext.workspaceVertex, writeLock) {
+    txn.withLock(workspace.toWorkspaceName, writeLock) {
```

Comment:
Not Future-y, as we really do want to execute the initial graph save immediately!